.github/workflows/bootstrapping_ci.yml

```diff
@@ -29,10 +33,11 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 10
-      - name: Build v
-        run: make
-      - name: v.c can be compiled and run with -os cross (bootstrapping works)
+      - name: Build V
+        run: make -j4
+      - name: Test bootstrapping (self-hosting compiler)
```

Comment:
The previous task name was more descriptive. Please restore it.

Reply:
I've updated the suggestion to combine them. Basically putting that shorter general concept description to the front.

Imho it could help making that mental assignment when thinking/talking about what V's bootstrapping actually does.

Please leave a note fully restoring it is preferred! The CI is quick here.